spot-client/src/common/css/modal.scss

```diff
@@ -14,6 +14,7 @@
 
     .modal-shroud {
         height: 100%;
+        pointer-events: fill;
```

Comment:
I'm not really a fan of this UX for this change. It looks odd to me that the other buttons on the page look clickable but actually are not.

Reply:
I agree, but I think in addition we should blur the background or apply some tinting. Otherwise the entire screen just looks like one big mess of buttons. What do you think?

E.g.:
![image](https://user-images.githubusercontent.com/9974975/58100429-e7e45300-7bdd-11e9-88b4-e5536b99c7f0.png)

Reply:
Especially on a mobile screen:

![image](https://user-images.githubusercontent.com/9974975/58100745-88d30e00-7bde-11e9-8874-73e93ed8a4f8.png)

Reply:
I remembered the UX design now. It’s intentional that the other buttons be clickable while the modal is displayed.

Reply:
Well, ok then